tests/issue/issue.cpp

```diff
@@ -480,3 +579,21 @@ TEST(Issue, isHint)
     e->setLevel(libcellml::Issue::Level::HINT);
     EXPECT_EQ(e->level(), libcellml::Issue::Level::HINT);
 }
+
+TEST(Issue, url)
+{
+    auto expectedUrl = "https://cellml-specification.readthedocs.io/en/latest/reference/formal_and_informative/specB01.html?issue=2.1.1";
```

Comment:
May want to replace with:
```
const std::string expectedUrl = "https://cellml-specification.readthedocs.io/en/latest/reference/formal_and_informative/specB01.html?issue=2.1.1";
```

Simply because this is how we do it in our other tests.

Reply:
We have a mixture of type definitions through the tests.